website/docs/r/vra_project.html.markdown

```diff
@@ -29,12 +29,45 @@ resource "vra_project" "this" {
   
   shared_resources = false
 
+  # Deprecated
```

Comment:
Can you also mention, "use administrator_roles instead"

Reply:
Done